app/code/Transoft/Blog/Model/DataProvider.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Model;
+
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Filesystem\Io\File;
+use Magento\Ui\DataProvider\AbstractDataProvider;
+use Transoft\Blog\Model\ModelRepository;
+use Transoft\Blog\Model\ResourceModel\Model\CollectionFactory;
+
+/**
+ * Blog model ui data provider
+ */
+class DataProvider extends AbstractDataProvider
+{
+    /**
+     * @var array
+     */
+    protected $_loadedData;
+
+    /**
+     * @var File $file
+     */
+    private $file;
+
+    /**
+     * @var ModelRepository
+     */
+    private $modelRepository;
+
+    /**
+     * @var SearchCriteriaBuilder $searchCriteriaBuilder
+     */
+    private $searchCriteriaBuilder;
+
+    /**
+     * Constructor.
+     *
```

Comment:
It's obvious. Please remove **Constructor** description. Fix for **all** files

Reply:
removed